src/Diagram.js

```diff
@@ -288,6 +288,42 @@ class Diagram extends Component {
         this._edges.unhighlightEdges(isSomeSelected);
     }
 
+    centerNode(name) {
+        const node = this._nodes.getNodeByName(name);
+        if (!this._zoomable || !this.isNodeOutOfView(node)) {
```

Comment:
Just wondering if _centerNode_ should center only if node is out of view. Maybe there could be some option or two separate methods -- one for forced centering, other for centering if out of view. What do you think?